po/ja-kana.po

```diff
@@ -0,0 +1,5137 @@
+#: js/blocks.js:4319
+#: js/block.js:720
+#: js/block.js:3892
+#: js/block.js:3901
+#: js/block.js:3912
+#: js/activity.js:2732
+#: js/analytics.js:475
+#: js/palette.js:681
+#: js/palette.js:690
+#: js/palette.js:699
+#: js/palette.js:708
+#: js/palette.js:1983
+#: js/palette.js:1994
+#: js/palette.js:2005
+#: js/palette.js:2016
+#: js/turtledefs.js:23
+#: js/basicblocks.js:1148
+#: js/basicblocks.js:1159
+#: js/basicblocks.js:1170
+#: js/basicblocks.js:2817
+#: js/basicblocks.js:2848
+#: js/basicblocks.js:2855
+#: js/basicblocks.js:2884
+#: js/basicblocks.js:2894
+#: js/basicblocks.js:2926
+#: js/basicblocks.js:2971
+#: js/basicblocks.js:2975
+#: js/basicblocks.js:2981
+#: js/macros.js:44
+#: js/blocks.js:784
+#: js/blocks.js:2385
+#: js/blocks.js:2386
+#: js/blocks.js:2578
+#: js/blocks.js:2784
+#: js/blocks.js:2940
+#: js/blocks.js:3514
+#: js/blocks.js:4319
+#.TRANS: a stack of blocks to run (an action to take)
+msgid "action"
+msgstr "アクション"
+
+#: js/block.js:723
+#: js/logo.js:1347
+#: js/logo.js:10186
+#: js/turtle.js:567
+#: js/turtle.js:568
+#: js/basicblocks.js:2963
+#: js/lilypond.js:586
+#: js/lilypond.js:737
+#: js/lilypond.js:775
+msgid "start"
+msgstr "スタート"
+
+#: js/block.js:726
+msgid "matrix"
+msgstr "ピッチ・じきぎょうれつ"
+
+#: js/block.js:729
+#: js/basicblocks.js:837
+#: plugins/rodi.rtp:328
+## is this for "status widget"? Maybe ステータスモニター;
+## Also 現状 http://jisho.org/word/%E7%8F%BE%E7%8A%B6
+msgid "status"
+msgstr "ステータス"
+
+#: js/block.js:732
+#: js/block.js:753
+#: js/palette.js:729
+#: js/turtle.js:409
+#: js/turtledefs.js:23
+#: js/basicblocks.js:996
+#: js/basicblocks.js:2199
+#: js/lilypond.js:569
+#: js/lilypond.js:727
+#: js/lilypond.js:765
+msgid "drum"
+msgstr "ドラム"
+
+#: js/block.js:735
+## ひらがな＝じょうぎ
+msgid "ruler"
+msgstr "ルーラー"
+
+#: js/block.js:738
+#: js/basicblocks.js:738
+#.TRANS: timbre is the character or quality of a musical sound
+## ひらがな＝ねいろ
+msgid "timbre"
+msgstr "ねいろ"
+
+#: js/block.js:741
+## I put a word that translates to "staircase"
+msgid "stair"
+msgstr "かいだん"
+
+#: js/block.js:744
+#: js/basicblocks.js:757
+#.TRANS: the speed at music is should be played.
+msgid "tempo"
+msgstr "テンポ"
+
+#: js/block.js:747
+#: js/basicblocks.js:1987
+#.TRANS: mode, e.g., Major in C Major
+## 旋法＝せんぽう；https://ja.wikipedia.org/wiki/%E3%83%A2%E3%83%BC%E3%83%89_(%E6%97%8B%E6%B3%95)
+msgid "mode"
+msgstr "モード（おんかい）"
+
+#: js/block.js:750
+msgid "slider"
+msgstr "スライダー"
+
+#: js/block.js:1662
+#: js/musicutils.js:1553
+#: js/musicutils.js:1934
+#.TRANS: the note names must be separated by single spaces
+#.TRANS: the note names must be separated by single spaces
+#.TRANS: the note names must be separated by single spaces
+## The spaces must not be "Japanese" spaces
+msgid "ti la sol fa mi re do"
+msgstr "シ ラ ソ ファ ミ レ ド"
+
+#: js/block.js:1858
+#: js/logo.js:718
+#: js/logo.js:8833
+#: js/palette.js:761
+#: js/blocks.js:1717
+#: js/blocks.js:2165
+## http://jisho.org/word/%E7%9C%9F 
+msgid "true"
+msgstr "そうだ"
```

Comment:
So keep in English as `true` and `false`?

We should also look at translations of other programming languages.

Reply:
I found the terms on this page: https://ja.wikipedia.org/wiki/If%E6%96%87#.E7.9C.9F.E5.81.BD.E5.80.A4

Reply:
I actually looked at Scratch and MakeCode... Scratch doesn't have the true and false block, and MakeCode uses " 真" and "偽" as translation. However, the translation is actually hard to express in Hiragana... "しん" and "ぎ" don't make sense... For now, we can go with "true" and "false" and fix them later once we come up with better words.

Reply:
my wife likes `ほんとうか` and `うそか`

so I will put it in for now (she is the real boss) :p

Reply:
For example, it would look like:

`ON`
![screenshot at 2018-08-29 13 29 57 on](https://user-images.githubusercontent.com/13454579/44804587-ba5b0380-ab8f-11e8-9414-0f85de7e5bbb.png)

or

`オン`
![screenshot at 2018-08-29 13 29 13](https://user-images.githubusercontent.com/13454579/44804627-e1193a00-ab8f-11e8-9530-a69494688493.png)

Reply:
Some web research results in:

`次のコードはチェックボックスの値を出力します。出力される値は「オン = True」「オフ = False」となります。`

https://www.petitmonte.com/excel/excel_vba_31.html

and

<pre>
'それぞれのチェックボックスにオン、オフ、未確定を設定する
CheckBox1.IsChecked = True	'オン
CheckBox2.IsChecked = False	'オフ
CheckBox3.IsChecked = Nothing	'未確定
</pre>
http://blog.hiros-dot.net/?page_id=3527